src/backend/base/langflow/components/processing/structured_output.py

```
@@ -177,7 +182,23 @@
             # handle empty or unexpected type case
             msg = "No structured output returned"
             raise ValueError(msg)
-        if len(output) != 1:
-            msg = "Multiple structured outputs returned"
+        if len(output) == 1:
+            return Data(data=output[0])
+        if len(output) > 1:
+            # Multiple outputs - wrap them in a results container
+            return Data(data={"results": output})
```

Ruff (0.11.9)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)